mypy/dmypy_server.py

```diff
@@ -254,9 +254,8 @@ def initialize_fine_grained(self, sources: List[mypy.build.BuildSource]) -> Dict
         self.fscache = FileSystemCache(self.options.python_version)
         self.fswatcher = FileSystemWatcher(self.fscache)
         self.update_sources(sources)
-        if not self.options.use_fine_grained_cache:
-            # Stores the initial state of sources as a side effect.
-            self.fswatcher.find_changed()
+        # Stores the initial state of sources as a side effect.
```

Comment:
Why was this changed to be executed unconditionally?

Reply:
So that we have accurate `fswatcher` cache information for files that we didn't read from the on-disk cache, now that we don't generate `CacheMeta`s